src/main/java/frc/robot/RobotContainer.java

```diff
@@ -96,6 +110,10 @@ public Command getAutonomousCommand() {
     return autoChooser.getSelected();
   }
 
+  public double getPivotEncoder() {
```

Comment:
I would make a method for this, I would just call pivot.getPosition() directly, you should just getEncoderValues to call the pivot encoder .getPosition().

Reply:
Actually, I see what you're doing, it's private. So there are two options:

1. Keep the function and just change getEncoderValues to use a .getPosition call instead
2. Just have a getPivot() which would return the subsystem

the choice is yours.